lib/xml_builder.ex

```diff
@@ -26,6 +29,9 @@ defmodule XmlBuilder do
   def doc(name, attrs, content),
     do: [:_doc_type | [element(name, attrs, content)]] |> generate
 
+  def comment(content),
+    do: element(nil, nil, content)
```

Comment:
Since its not a real element, would be ideal if we dont call `element/3`, maybe we can do something like we did with `:_doc_type`, like have a key `:_comment` to identify comments. thoughts?